SetupDataPkg/Tools/KnobService.py

```diff
@@ -1289,8 +1310,15 @@ def main():
                     sys.stderr.write('Invalid profile names, should be 2-character for all entries.\n')
                     return -1
 
+            if known_args.profile_ids is not None:
+                # Do some minimal sanity checks
+                ids = known_args.profile_names.split(",")

```

Comment:
Since we only have a UINT8 for this PID, can we please verify the incoming values are in valid range?

Reply:
Added a check to validate each profile id is within valid range or not.